core/src/main/java/org/commonjava/indy/core/content/DefaultContentManager.java

```diff
@@ -742,6 +742,13 @@ public HttpExchangeMetadata getHttpMetadata( final StoreKey key, final String pa
         return readExchangeMetadata( meta );
     }
 
+    @Override
+    public boolean exists(ArtifactStore store, String path)
+        throws IndyWorkflowException
+    {
+        return downloadManager.exists(store, path);
```

Comment:
Please add a //TODO to add content generation handling here, for things like merged metadata, checksum files, etc. We will have to tackle those soon.